crates/ide-completion/src/tests/expression.rs

```diff
@@ -652,6 +652,26 @@ fn main() {
     );
 }
 
+#[test]
+fn bound_type_in_impl_trait() {
+    check_empty(
+        r"
+struct A;
+trait B {
+    type O: ?Sized;
+}
+impl B for A {
+    $0
+}
+",
+        expect![[r"
+    ta type O =
+    kw crate::
+    kw self::
+    "]],
+    )
+}
```

Comment:
This isn't an expression test, this test should go into `item_list.rs`

Reply:
I would remove this test case as it seems a bit redundant